src/App.tsx

```diff
@@ -2,7 +2,9 @@ import React, { useEffect } from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { Provider as PaperProvider } from 'react-native-paper';
 import SplashScreen from 'react-native-splash-screen';
+import { Provider as StoreProvider } from 'react-redux';
 
+import { store } from './store/store';
```

Comment:
Let's rename store.ts to index.ts
```suggestion
import { store } from 'src/store';
```